packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -88,12 +89,15 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
                 <FocusZone direction={FocusZoneDirection.horizontal}>
                   <div className={this._classNames.chartTitle}>
                     {points!.chartTitle && (
-                      <div
-                        className={this._classNames.chartDataText}
```

Comment:
what does this element correspond to

Reply:
This div is for the chart title (the left side text above the bar)
![image](https://user-images.githubusercontent.com/110246001/195319597-e7105514-5485-4356-b375-1e6c99db2058.png)